lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -62,7 +63,7 @@ function resolveLocalOrCwd(payloadPath) {
  * @return {ExpectedLHR}
  */
 function runLighthouse(url, configPath, isDebug) {
-  isDebug = isDebug || Boolean(process.env.SMOKEHOUSE_DEBUG);
+  isDebug = isDebug || Boolean(process.env.LH_SMOKE_DEBUG);
```

Comment:
doesn't debug also save the LHR? Up to y'all if you want that :)

Reply:
nah. just writing to disk is good enough for me. verbose shouldn't be _too_ verbose